upgrade.php

```diff
@@ -438,11 +438,11 @@ function color($string, $color = '')
     }
 
     // Add new column and index to maillog table
-    echo pad(' - Add maillog_id field and primary key to `maillog` table');
-    if (true === check_column_exists('maillog', 'maillog_id')) {
+    echo pad(' - Add maillog_id field, rblspamreport and primary key to `maillog` table');
+    if (true === check_column_exists('maillog', 'maillog_id', 'rblspamreport')) {
```

Comment:
I'm pretty sure that this won't work (3 params?).
Also, in this case,  it's better to add single column: if migration from RC4 to RC5 has already been done this will not work